### PR TITLE
implement SNO gallery

### PR DESCRIPTION
**Pull request by GreenCappuccino**
Per #23 `gallery` shortcode is now implemented on same section as `ngg`!

Screenshot_1633845917

However, this implementation does not fetch author names.... something that has to be implemented later, and is blocked by the API.

### CODE REVIEW

File: lib/ui/pages/posts_page.dart [hunk 1]
```diff
@@ -25,10 +24,9 @@ class PostsPageState extends State<PostsPage> with TickerProviderStateMixin {
           Tab(child: Text("All")),
           ...tabCategories.map((e) => Tab(child: Text(e.name))).toList()
         ]),
-        child: Flexible(
-            child: TabBarView(controller: tabController, children: [
+        child: TabBarView(controller: tabController, children: [
```
Comment (GreenCappuccino) [inline lib/ui/pages/posts_page.dart:27]: Small fix to prevent exception from occurring related to `Flexible`

File: lib/main.dart [hunk 2]
```diff
@@ -16,7 +17,7 @@ import 'package:wordpress_api/wordpress_api.dart';
 void main() async {
   Logger.root.level = Level.INFO;
   Logger.root.onRecord.listen((record) {
-    print(
+    debugPrint(
```
Comment (GreenCappuccino) [inline lib/main.dart:20]: `debugPrint` will not attempt to print on production builds, as opposed to `print`

**Comment by GreenCappuccino** [comment]
@daniel-n6 I recommend taking a quick look at this and pulling changes before continuing work on #18 .

**Review by daniel-n6 — APPROVED** [review]
Works, although we may need enhancement later on to improve how the gallery looks

**Comment by GreenCappuccino** [comment]
Yep, as I understand you're working on #18 ? This actually uses the same code to render the gallery as for NextGEN galleries... so resolving that will do so for both types of galleries.

**Pull request merged**